packages/preact-router/src/SafeFragment.tsx

```
@@ -0,0 +1,13 @@
+import { createElement as h, Fragment } from 'preact'
```

ESLint

[error] 'h' is defined but never used.

(unused-imports/no-unused-imports)